packages/network/test/unit/MetricsContext.test.ts

```diff
@@ -217,22 +217,22 @@ describe('metrics', () => {
         metricOne.record('c', 208)
         metricTwo.record('a', 39)
         metricTwo.set('b', 100)
-        jest.advanceTimersByTime(1000)
+        jest.advanceTimersByTime(2000)
 
         const rep3 = await context.report()
         expect(rep3.metrics).toEqual({
             metricOne: {
                 a: 15,
                 b: 666,
                 c: {
-                    rate: 308,
+                    rate: 308 / 4,
```

Comment:
Where does the divider come from?

Reply:
Elapsed full seconds: there are `advanceTimersByTime` calls for 0.2s + 2s + 2s. 

Refactored this to calculate the value from `Date.now()`